fala/templates/adviser/results.html

```diff
@@ -23,10 +23,7 @@
   {% if tailored_results %}
     <div class="govuk-grid-column-full">
       <form action="/check" method="GET" novalidate>
-        {% for category in form.categories %}
-          <input type="hidden" name="categories" value="{{ category }}">
-        {% endfor %}
-        <a href="/check?categories={% for category in form.categories %}{{ category }}{% if not loop.last %},{% endif %}{% endfor %}" class="govuk-back-link govuk-!-margin-bottom-5 govuk-!-display-none-print" id="back_link">
+        <a href="/check?categories={{category_code}}{% if sub_category_code %}&sub-category={{ sub_category_code }}{% endif %}" class="govuk-back-link govuk-!-margin-bottom-5 govuk-!-display-none-print" id="back_link">
```

Comment:
https://design-system.service.gov.uk/components/back-link/ I think this can be a Jinja2 component here for the back button. I know it's a left over from someone else's work.

Reply:
Thank you for acknowledging this change is not part of this ticket 🙂

I've now updated to a Jinja2 component.